assets/src/edit-story/elements/video/frame.js

```diff
@@ -14,14 +14,68 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import styled from 'styled-components';
+
 /**
  * Internal dependencies
  */
 import StoryPropTypes from '../../types';
 import MediaFrame from '../media/frame';
+import { elementFillContent } from '../shared';
+import { useStory } from '../../app/story';
+import { ReactComponent as PlayIcon } from '../../icons/play.svg';
+import { ReactComponent as PauseIcon } from '../../icons/pause.svg';
+
+const Play = styled(PlayIcon)`
+  width: 48px;
+  height: 48px;
+`;
+const Pause = styled(PauseIcon)`
+  width: 48px;
+  height: 48px;
+`;
```

Comment:
Maybe the sizing should follow the element size somewhat? For a tiny video, this is a huge button!
![Screen Shot 2020-03-23 at 9 07 15 PM](https://user-images.githubusercontent.com/637548/77377731-6522ac00-6d4a-11ea-97f5-14c260d7fb63.png)

Reply:
@samitron7 any recommendations? (min. size?)

Reply:
I calculated 14% min. That should give us a decent size button without having to cover too much of the video. Let me if you all disagree.

Reply:
14% is fine. But I think you mean "max", right?

Reply:
@merapi Ok, let's set here 14% as the max size.

Reply:
That's right, we both meant max, but it's 14% of width or height of the container?